pkg/table-router/router.go

```diff
@@ -123,7 +123,7 @@ func (r *Table) RemoveRule(rule *TableRule) error {
 
 // Route routes schema/table to target schema/table
 // don't support to route schema/table to multiple schema/table
-func (r *Table) Route(schema, table string) (string, string, error) {
+func (r *Table) Route(schema, table string) (targetSchema string, targetTable string, matched bool, err error) {
```

Comment:
it would affect `syncer`/`loader`/`dm`, I wouldn't like to change it like this, if you need to filter `test2`, just give a bw list

Reply:
dm can just ignore the return value `matched`

Reply:
it's very strange, some one would think you should compare `test2` and `test2`